tests/integration/components/sl-calendar-test.js

```diff
@@ -0,0 +1,1461 @@
+        '2020-2029',
+        'The current decade is set correctly'
+    );
+
+    this.$( '>:first-child .active' ).click();
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        currentYear,
+        'The current year is set correctly'
+    );
+});
+
+test( 'Navigating Forward by Month Crosses to Next Year', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear', currentYear );
+
+    const currentMonth = 12;
+    this.set( 'currentMonth', currentMonth );
+
+    this.render( hbs`
+        {{sl-calendar
+            currentYear=currentYear
+            currentMonth=currentMonth
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'December ' + currentYear,
+        'The current month is set correctly'
+    );
+
+    this.$( '>:first-child .next' ).click();
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'January ' + ( currentYear + 1 ),
+        'The next month is in the next year'
+    );
+});
+
+test( 'Navigating Backward by Month Crosses to Previous Year', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear', currentYear );
+
+    const currentMonth = 1;
+    this.set( 'currentMonth', currentMonth );
+
+    this.render( hbs`
+        {{sl-calendar
+            currentYear=currentYear
+            currentMonth=currentMonth
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'January ' + currentYear,
+        'The current month is set correctly'
+    );
+
+    this.$( '>:first-child .prev' ).click();
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'December ' + ( currentYear - 1 ),
+        'The previous month is in the previous year'
+    );
+});
+
+test( 'All Twelve Months are Displayed in Order', function( assert ) {
+
+    this.render( hbs`
+        {{sl-calendar
+            viewMode="months"
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child .month' ).text().trim(),
+        'JanFebMarAprMayJunJulAugSepOctNovDec',
+        'Twelve months are listed in order'
+    );
+});
+
+test( 'Twelve Years are Displayed in Order', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear', currentYear );
+
+    this.render( hbs`
+        {{sl-calendar
+            viewMode="years"
+            currentYear=currentYear
+        }}
+    ` );
+
+    let yearsSelection = this.$( '>:first-child' ).find( 'td' ).text().trim();
+    yearsSelection = yearsSelection.replace( /\s+/g, '' );
+
+    assert.equal(
+        yearsSelection,
+        '201920202021202220232024202520262027202820292030',
+        'Twelve years are listed in order'
+    );
+});
+
+// ******************************
+// Start of Dual Component Tests
+// ******************************
```

Comment:
Change to

```
// -------------------------------------------------------------------------
// Start of Dual Component Tests
```

This follows the same convention as https://github.com/softlayer/ember-style-guide#file-structure-1 as well as allows for a group of tests to easily be block-commented out without this comment block interacting with this intent.